### PR TITLE
Updated remaining Nonreference RomanCal Schemas

### PR DESCRIPTION
**Pull request by PaulHuwe**
This PR is to update the remaining schemas to the latest in RomanCAL. The affected schemas are: aperture, basic, ephemeris, exposure, guidestar, observation, pixelarea, and visit

Tests: Passes standard tests

Docs: Unneeded

Ticket: [#46]

### CODE REVIEW

File: src/rad/resources/schemas/observation-1.0.0.yaml [hunk 1]
```diff
@@ -178,15 +178,22 @@ properties:
     archive_catalog:
       datatype: nvarchar(50)
       destination: [ScienceCommon.ma_table_name]
+  survey:
+    title: Flat Field Step
```
Comment (perrygreenfield) [inline src/rad/resources/schemas/observation-1.0.0.yaml:182]: Flat Field Step? Did this get copied into this attribute by mistake?

**Review by perrygreenfield — COMMENTED** [review]
only one minor issue.

File: src/rad/resources/schemas/observation-1.0.0.yaml (hunk 1 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/observation-1.0.0.yaml]: Corrected.

**Review by perrygreenfield — APPROVED** [review]
LGTM

**Pull request merged**